whats_fresh/whats_fresh_api/templates/entry.html

```diff
@@ -3,38 +3,47 @@
 {% block content %}
 <div class='entry'>
     <a href='{% url 'list-vendors-edit' %}'>
-        <h4>Vendors</h4>
+        <!-- <h4>Vendors</h4> -->
+        <h4>{{ theme|get_fieldname:'vendors' }}</h4>
```

Comment:
No, sorry, I think I accidentally confirmed that when what I meant is that even when we're extending we still need to reload it.